spec/graphql/mutations/groups/create_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+RSpec.describe Mutations::Groups::Create, type: :request do
+  let!(:user) { create :user, :owner}
+  let!(:token) { generate_jwt_test_token(user) }
+
+  let!(:variables) do
+    {
+      input: {
+        groupAttributes: {
+          name: 'Group 1',
+          description: 'Group 1 Test',
+          slots: 5,
+          country: 'Canada',
+          region: 'West',
+          city: 'Vancouver',
+          streetAddress: 'Fake Street',
+          postCode: '123 456',
+          private: false
+        }
+      }
+    }
+  end
+    
+  let!(:create_group_query) do
+    <<-GQL
+      mutation createGroup ($input: CreateGroupInput!) {
+        createGroup (input: $input){
+          group {
+            id,
+            name,
+            description,
+            slots,
+            country,
+            region,
+            city,
+            streetAddress,
+            postCode,
+            private,
+            attend
+          }
+          errors
+        }
+      }
+    GQL
+  end
+
+  describe 'with valid variables' do
+    it 'creates the group' do
+      expect do
+        post '/graphql', params: { query: create_group_query, variables: variables.to_json }, headers: { 'Authorization' => "Bearer #{token}" }
```

Comment:
I have realize we were using [execute_and_parse_graphql_response](https://github.com/vancityjun/focus-backend/blob/e9b811c73942ff85b09590bcd0919ca095d4ec0a/spec/support/graphql/response_parser.rb#L3) before
and it has been changed to this since user update commit is there any specific reason? if not let's use the method you've defined instead

Reply:
yeah. You're right I will change this using `execute_and_parse_graphql_response`.